### PR TITLE
1928 refactor out server GSL

### PR DESCRIPTION
**Pull request by dionjwa**
This PR:

 - Removes all references to the GSL standalone server.
 - Calls the new GC server route `/compute` method to perform the GSL compute.
 - Downloads are not stored on the server, they're stored in memory only (since computes take seconds, they can be trivially recomputed when needed).

Otherwise the extension works exactly as before (to the user).

There is some config from the server copied to the client, but the compute command is hard-coded for now, this will be cleaned up when we have the primers working. 

Relates to: https://github.com/autodesk-lifesciences/genetic-constructor/pull/1931

### CODE REVIEW

File: src/components/editor/CodeEditorLayout.js [hunk 1]
```diff
@@ -334,33 +361,42 @@ export default class CodeEditorLayout extends Component {
...
+        });
+    }
   }
 
   /**
    * Download a file depending on the item clicked
    * @param {MouseEvent} click event
    */
   doDownload(key) {
+    console.log('doDownload key', key);
```
Comment (dhylbert) [inline src/components/editor/CodeEditorLayout.js]: I don't object to this log statement, but just calling it out incase you wanted to remove it.

File: src/components/editor/CodeEditorLayout.js [hunk 2]
```diff
@@ -185,10 +200,13 @@ export default class CodeEditorLayout extends Component {
...
-      .then((data) => {
-        this.onDownloadMenuSettingsChange(data);
-      });
+    const projectId = this.getProjectId();
+    const files = (gslState[projectId] && gslState[projectId].outputs) || {};
+    this.onDownloadMenuSettingsChange(files);
+    // compiler.getAvailableDownloadList(this.getProjectId())
+    //   .then((data) => {
+    //     this.onDownloadMenuSettingsChange(data);
+    //   });
```
Comment (dhylbert) [inline src/components/editor/CodeEditorLayout.js]: Worth just deleting these commented-out lines?

File: src/behavior/compiler/client.js [hunk 3]
```diff
@@ -34,23 +34,38 @@ export const writeProjectFile = (projectId, code, fileName = 'project.gsl') =>
...
+mono /gslc/bin/Gslc.exe --lib /gslc/gslc_lib --flat /outputs/gslOutFlat.txt --json /outputs/gslOut.json --primers /outputs/gslOut.primers.txt --ape /outputs gslOut --cm /outputs gslOut /inputs/project.gsl
+`;
+
   const payload = {
-    code,
-    projectId,
-    args,
-    extension,
+    CreateContainerOptions: {
+      Image: 'docker.io/dionjwa/gslc:1a5fe0e6', // TODO: move this to quay.io
```
Comment (dionjwa) [inline src/behavior/compiler/client.js]: I've made a ticket for next sprint: https://zube.io/autodesk/genome-designer/c/1980

File: src/components/editor/CodeEditorLayout.js (hunk 2 above)
Comment (dionjwa) [inline src/components/editor/CodeEditorLayout.js]: Yes!

File: src/components/editor/CodeEditorLayout.js (hunk 1 above)
Comment (dionjwa) [inline src/components/editor/CodeEditorLayout.js]: Removed.

**Pull request merged**